### ISSUE TITLE
Cannot pass script args in PowerShell

### ISSUE THREAD

**Issue by adamralph** [scriptcs-contrib/svm#70]
```
C:\Adam\Apps\Scripts> scriptcs .\ubs_export.csx -- export.csv
Unexpected unnamed argument: export.csv
```

This is because `scriptcs` is resolving to `scriptcs.ps1` and `--` doesn't work with PowerShell because, er,  it has some kind of special meaning?

If I force use of the batch file `scriptcs.cmd .\ubs_export.csx -- export.csv` then it works just fine.

I wonder if it may be better to call `scriptcs.ps1` something else, so that PS resolves `scriptcs` to `scriptcs.cmd`?

**Comment by paulbouwer** [comment]
Thanks @adamralph ! 

I've always run via the `Windows Console` where the `.ps1` file does not get resolved but rather the `.cmd`. This was a subtlety that I had not even considered before.

I've got a fix teed up for this.

**Comment by ryanrousseau** [comment]
I found a work around for passing arguments in powershell until the issue is resolved.

If you use two sets of "--", the arguments are passed correctly.

![image](https://cloud.githubusercontent.com/assets/1571799/10711060/1db3a218-7a36-11e5-81c7-b7edfe1061ed.png)

![image](https://cloud.githubusercontent.com/assets/1571799/10711062/2c0ac01c-7a36-11e5-89e3-8da1629ba339.png)

**Comment by adamralph** [comment]
Thanks @ryanrousseau, good to know!

**Comment by paulbouwer** [comment]
Fixed in bf28290314531074aef75ceb74e1b2bf10580e34 - finally getting around to releasing now. Thanks @adamralph and @ryanrousseau.
